From 9f1bf5e004ccd81424f802cef48072aa9a69728e Mon Sep 17 00:00:00 2001
From: Zhenggen Xu <zxu@linkedin.com>
Date: Thu, 3 Dec 2020 11:43:53 -0800
Subject: [PATCH] [FRR log] Stop FRR log inside the docker

The frr.log file inside docker could grow without limit today.
With the fix, we will stop the logging inside docker to remove
the duplication since we already log to syslog at host side.

Signed-off-by: Zhenggen Xu <zxu@linkedin.com>
---
 tools/etc/rsyslog.d/45-frr.conf | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/tools/etc/rsyslog.d/45-frr.conf b/tools/etc/rsyslog.d/45-frr.conf
index ff7cd4831..608012c29 100644
--- a/tools/etc/rsyslog.d/45-frr.conf
+++ b/tools/etc/rsyslog.d/45-frr.conf
@@ -2,23 +2,6 @@
 # to /var/log/frr/frr.log, then drops the message so it does
 # not also go to /var/log/syslog, so the messages are not duplicated
 
-$outchannel frr_log,/var/log/frr/frr.log
-if  $programname == 'babeld' or
-    $programname == 'bgpd' or
-    $programname == 'eigrpd' or
-    $programname == 'frr' or
-    $programname == 'isisd' or
-    $programname == 'ldpd' or
-    $programname == 'nhrpd' or
-    $programname == 'ospf6d' or
-    $programname == 'ospfd' or
-    $programname == 'pimd' or
-    $programname == 'ripd' or
-    $programname == 'ripngd' or
-    $programname == 'watchfrr' or
-    $programname == 'zebra'
-    then :omfile:$frr_log
-
 if  $programname == 'babeld' or
     $programname == 'bgpd' or
     $programname == 'eigrpd' or
-- 
2.18.0

